tools/sotn-debugmodule/Makefile

```diff
@@ -19,21 +19,25 @@ CC_FLAGS        += -c -march=r3000 -fno-stack-protector -mtune=r3000 -EL -G0 -O2
 CPP_FLAGS       += -I$(INCLUDE_DIR) -undef
 CPP_FLAGS       += -D_internal_version_$(VERSION) -Dmips -D__OPTIMIZE__ -D__mips__ -D__mips -Dpsx -D__psx__ -D__psx -D_PSYQ -D__EXTENSIONS__ -D_MIPSEL -DNO_LOGS -D_LANGUAGE_C -DLANGUAGE_C
 
-all: build
+all: clean_symbols build
 build: $(BUILD_DIR)/$(MODULE_NAME).bin
 clean:
 	rm -f $(BUILD_DIR)/$(MODULE_NAME).*
 	rm -rf $(BUILD_DIR)/tools/$(MODULE_NAME)
+	rm -f $(CONFIG_DIR)/symbols.$(VERSION).ld.txt
 format:
 	clang-format -i $$(find . -type f -name "*.c")
 	clang-format -i $$(find . -type f -name "*.h")
+clean_symbols:
+	@echo "Processing: $(CONFIG_DIR)/symbols.$(VERSION).txt -> $(CONFIG_DIR)/symbols.$(VERSION).ld.txt"
+	grep -o '^[^/]*' "$(CONFIG_DIR)/symbols.$(VERSION).txt" > "$(CONFIG_DIR)/symbols.$(VERSION).ld.txt"
 
 $(BUILD_DIR)/$(MODULE_NAME).bin: $(BUILD_DIR)/$(MODULE_NAME).elf
 	$(OBJCOPY) -O binary $< $@
 	printf '\x00' | dd of=$@ bs=1 seek=40959 count=1 conv=notrunc
 
 $(BUILD_DIR)/%.elf: $(BUILD_DIR)/tools/%/inject.o $(BUILD_DIR)/tools/%/debugmode.o $(BUILD_DIR)/tools/%/menu.o $(BUILD_DIR)/tools/%/fontmanager.o $(BUILD_DIR)/tools/%/debug_flags.o $(BUILD_DIR)/tools/%/entity_spawn.o $(BUILD_DIR)/tools/%/sfx_player.o $(BUILD_DIR)/tools/%/flag_checker.o
-	$(LD) -o $@ -Map $*.map -T $*.ld -T $(CONFIG_DIR)/symbols.$(VERSION).txt \$^
+	$(LD) -o $@ -Map $*.map -T $*.ld -T $(CONFIG_DIR)/symbols.$(VERSION).ld.txt \$^
```

Comment:
`mipsel-linux-gnu-ld` doesn't like symbols files with comments, unlike splat. 
I've tried to keep this self-contained, copies to a new file and strips the comments.

Reply:
@Xeeynamo this didn't seem to work for me, get a whole heap of undefined references. Even if I copy over the whole `symbols.us.txt` file into `undefined_syms.us.txt` it still fails compiling DRA